packages/espressocash_client/lib/src/espressocash_client.dart

```diff
@@ -18,6 +18,20 @@ const defaultBaseUrl = 'grpc.espressocash.com';
 typedef SignRequest = Future<String> Function(Iterable<int> data);
 
 class EspressoCashClient {
+  factory EspressoCashClient({
+    String? baseUrl,
+    int? port,
+    required SignRequest sign,
+    String? walletAddress,
+    bool secure = true,
+  }) => EspressoCashClient._(
+    baseUrl: baseUrl ?? defaultBaseUrl,
+    port: port,
+    sign: sign,
+    walletAddress: walletAddress,
+    secure: secure,
+  );
+
   factory EspressoCashClient.anonymous({String? baseUrl, int? port, bool secure = true}) =>
```

Comment:
Do we need `anonymous` constructor at all?

Reply:
in current code it is used here: https://github.com/espresso-cash/espresso-cash-public/blob/6c2b38f05f53d895e733665edd7f8de12118f3a9/packages/espressocash_app/lib/features/dynamic_links/data/dynamic_links_client.dart#L8